### PR TITLE
feat: for visitors page

### PR DESCRIPTION
**Pull request by FrantisekMichalSebestyen**

### CODE REVIEW

File: resources/js/ForVisitors.vue [hunk 1]
```diff
@@ -0,0 +1,149 @@
+<template>
+    <div class="mx-auto container px-5 h-60 flex items-end">
+        <h1 class="font-display text-4xl md:text-5xl text-blue-dark tracking-wide max-w-48 md:max-w-full">
+            {{ $t("For visitors") }}
+        </h1>
+    </div>
+    <div class="container mx-auto px-6 py-12 md:pt-16">
+        <div class="mr-auto max-w-[800px] mt-4">
+            <section class="mb-4">
```
Comment (igor-kamil) [inline resources/js/ForVisitors.vue:9]: there should be bigger space under the section on desktop 
Screenshot 2024-03-19 at 23 38 30

in design:

Screenshot 2024-03-19 at 23 40 15

**Review by igor-kamil — COMMENTED** [review]
looks very nice both on desktop and mobile. 
we need to include translations before releasing + fix the spacing between sections. 
almost there.

**Comment by FrantisekMichalSebestyen** [comment]
Ja by som texty pripomienkoval: 3 rôzne formáty času v angličtine.
`Časy performancií počas preview`. Sa mi nezdá, ale tak hodím to tam.

File: resources/js/ForVisitors.vue [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+                />
+                <div class="flex flex-col md:flex-row justify-between w-full py-4">
+                    <span class="font-display text-2xl">{{ $t("18.4") }}</span>
+                    <span class="text-xl">{{ formatTime("1700") }} - {{ formatTime("1900") }} {{ $t("(TBA)") }}</span>
+                </div>
+                <Separator
+                    class="bg-white opacity-15 data-[orientation=horizontal]:h-px data-[orientation=horizontal]:w-full mb-4"
+                />
+                <div class="flex flex-col md:flex-row justify-between w-full py-4">
+                    <span class="font-display text-2xl">{{ $t("19.4") }}</span>
```
Comment (igor-kamil) [inline resources/js/ForVisitors.vue:40]:
```suggestion
                    <span class="font-display text-2xl">{{ $t("19.4.") }}</span>
```

File: resources/js/ForVisitors.vue [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+                <Separator
+                    class="bg-white opacity-15 data-[orientation=horizontal]:h-px data-[orientation=horizontal]:w-full mb-4"
+                />
+            </section>
+            <section class="mb-4 mt-6">
+                <h2 class="pb-2 opacity-60">
+                    {{ $t("Schedule of performances during preview") }}
+                </h2>
+                <div class="flex flex-col md:flex-row justify-between w-full py-4">
+                    <span class="font-display text-2xl">{{ $t("17.4") }}</span>
```
Comment (igor-kamil) [inline resources/js/ForVisitors.vue:26]:
```suggestion
                    <span class="font-display text-2xl">{{ $t("17.4.") }}</span>
```

File: resources/js/ForVisitors.vue [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+                    <span class="text-xl">{{ formatTime("1400") }}</span>
+                </div>
+                <Separator
+                    class="bg-white opacity-15 data-[orientation=horizontal]:h-px data-[orientation=horizontal]:w-full mb-4"
+                />
+                <div class="flex flex-col md:flex-row justify-between w-full py-4">
+                    <span class="font-display text-2xl">{{ $t("18.4") }}</span>
```
Comment (igor-kamil) [inline resources/js/ForVisitors.vue:33]:
```suggestion
                    <span class="font-display text-2xl">{{ $t("18.4.") }}</span>
```

File: resources/js/ForVisitors.vue [hunk 5]
```diff
@@ -0,0 +1,173 @@
...
+        </h1>
+    </div>
+    <div class="container mx-auto px-6 py-12 md:pt-16">
+        <div class="mr-auto max-w-[800px] mt-4">
+            <section class="mb-4">
+                <h2 class="pb-2 opacity-60">
+                    {{ $t("Opening") }}
+                </h2>
+                <div class="flex flex-col md:flex-row justify-between w-full py-4">
+                    <span class="font-display text-2xl">{{ $t("18.4") }}</span>
```
Comment (igor-kamil) [inline resources/js/ForVisitors.vue:14]:
```suggestion
                    <span class="font-display text-2xl">{{ $t("18.4.") }}</span>
```

**Review by igor-kamil — CHANGES_REQUESTED** [review]

**Pull request merged**